config/locales/en.yml

```
@@ -34,3 +34,4 @@ en:
   activity:
     user_logged_in: "User logged in with the next attributes:\n First name - %{first_name},\n Last name - %{last_name}\n"
     user_registered: "New user registered with the next attributes:\n First name - %{first_name},\n Last name - %{last_name}\n"
+    user_updated_password: "User updated password"
```

```suggestion
    user_reset_password: "User reset password"
```

Should it be a `reset` verb? User can update password without reset it through update profile action. And to distinguish them it would be better to name as `reset` I think.